Library/Homebrew/utils/curl.rb

```diff
@@ -12,11 +12,14 @@ def curl_executable
 end
 
 def curl_args(*extra_args, show_output: false, user_agent: :default)
-  args = [
-    curl_executable.to_s,
-    "-q", # do not load .curlrc (must be the first argument)
-    "--show-error",
-  ]
+  args = [curl_executable.to_s]
+
+  # do not load .curlrc unless requested (must be the first argument)
+  if ENV["HOMEBREW_CURLRC"]
```

Comment:
Isn't this backwards? Now we will still load curlrc by default.

Reply:
@stefansundin It is indeed, I'm a moron! Fixed in https://github.com/Homebrew/brew/pull/4045, thanks for noticing!